src/components/Button/index.tsx

```diff
@@ -386,6 +386,13 @@ const FwdButton = ({
     return 'button'
   }, [disabled, to, href, download])
 
+  const ariaLabel = useMemo(() => {
+    if (props['aria-label']) return props['aria-label']
+    if (typeof children === 'string') return children
+
+    return ''
```

Comment:
No need to specify an aria label if not needed, meaning it depends on the usage

Reply:
why we do those tickets ? your point is correct, but this mean we need to force use of aria label if text is not specified or so